integration_tests/e2e/prisoner-details.cy.ts

```diff
@@ -1,64 +1,76 @@
 import LogPrisonerDetailsPage from '../pages/logPrisonerDetails'
 import Page from '../pages/page'
 
-context('Log Prisoner Details Page', () => {
-  let page: LogPrisonerDetailsPage
+const appTypes = [
```

Comment:
You could pull this in from `integration_tests/fixtures/applicationTypes.json` to save duplication?